spec/widgets/time-series/histogram-view.spec.js

```diff
@@ -131,6 +136,19 @@ describe('widgets/time-series/histogram-view', function () {
     });
   });
 
+  describe('_onForceResize', function () {
+    it('should call _chartView.onWindowResize function', function () {
+      this.view._chartView = {
```

Comment:
I think you can save a few of lines with `this.view._chartView = jasmine.createSpyObj('chartView', ['onWindowResize']);`.

Reply:
Thanks! :+1: